x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -1065,42 +1065,40 @@ export function overridePackageInputs(
     }),
   ];
 
-  for (const override of inputsOverride) {
+  for (const update of inputsUpdated) {
     // Preconfiguration does not currently support multiple policy templates, so overrides will have an undefined
```

Comment:
is this comment relevant for the updatePackageInputs method?

Reply:
No it's not, going to update the comment 👍